app/src/main/res/layout/fragment_single_post.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Make this view look like edit and create post in the size of the image and post content area